Documentation/building/unix-instructions.md

```diff
@@ -1,5 +1,8 @@
 Building CoreFX on FreeBSD, Linux and OS X
 ==========================================
+
+Build instructions for FreeBSD can be found [here](freebsd-instructions.md)
```

Comment:
FreeBSD isn't supported, whereas Linux and macOS are.  Seems like "Building CoreFX on FreeBSD, Linux, and OSX" should instead be "Building CoreFX on Linux and macOS", and then this line should be in its own section later.

Reply:
I'll move it to its own section at the end of this document.